frontend/src/pages/dashboard/jios/edit/index.tsx

```diff
@@ -35,11 +36,16 @@ export default function JiosEdit() {
       enqueueSnackbar('Failed to get Jio.', { variant: 'error' });
     },
   });
+
+  const navigateOut = () => {
```

Comment:
I have seen similar function in create/. Perhaps we should extract them out ?

Reply:
Agree.

Reply:
Actually I think we can just extract out the constant instead. Can't really meaningfully reduce LoC since with are using a function from a hook.